### ISSUE TITLE
mxnet.metric.EvalMetric.get_config docs error

### ISSUE THREAD

**Issue by aaronmarkham** [apache/incubator-mxnet#13009]
## Description
Sphinx throws an error when generating docs for `mxnet.metric.EvalMetric.get_config`.

## Error
```
/home/ubuntu/incubator-mxnet/python/mxnet/metric.py:docstring of mxnet.metric.EvalMetric.get_config:1: WARNING: Inline strong start-string without end-string.
```

**Comment by frankfliu** [comment]
@mxnet-label-bot [Doc]

**Comment by vrakesh** [comment]
issue fixed in PR #13215 , good to close
